### PR TITLE
Join scheduler and annotator

### PR DESCRIPTION
**Pull request by mmucek95**

### CODE REVIEW

File: tests/scheduler/test_server.py [hunk 1]
```diff
@@ -1,59 +1,67 @@
...
-        output = annotating_service._mutate_pod()
-        assert output.json == {
-                "response": {
-                    "allowed": True,
-                    "status": {"message": 'Patching pod'},
-                    "patch": base64.b64encode(str(patch).encode()).decode(),
-                    "patchtype": "JSONPatch",
-                }
-            }
+<<<<<<< HEAD:tests/scheduler/test_server.py
```
Comment (Creatone) [inline tests/scheduler/test_server.py:1]: Unnecessary

File: wca/scheduler/server.py [hunk 2]
```diff
@@ -53,10 +63,84 @@ def reschedule_interval(self, interval: Numeric(0, 60)):
...
+                    "allowed": True,
+                    "status": {"message": message},
+                    "patch": base64.b64encode(str(patch).encode()).decode(),
+                    "patchtype": "JSONPatch",
+                }
+            }
+        )
+
+    def _get_memory_type(self, wss: str, rss: str):
+        ratio = float(wss) / float(rss) * 100
```
Comment (Creatone) [inline wca/scheduler/server.py:81]: Possible `ZeroDivisionError`

**Review by Creatone — CHANGES_REQUESTED** [review]

File: tests/scheduler/test_server.py (hunk 1 above)
Comment (mmucek95) [reply tests/scheduler/test_server.py]: Removed

File: wca/scheduler/server.py (hunk 2 above)
Comment (mmucek95) [reply wca/scheduler/server.py]: Fixed

**Review by Creatone — APPROVED** [review]

**Pull request merged**